frontend/src/app/constants/constants.d.ts

```diff
@@ -594,6 +594,7 @@ interface PersonUpdate extends Address {
   county: string;
   country: string;
   email: string;
```

Comment:
remove this fields, we dont need it in the frontend

Reply:
I reckon we might as well. I originally included this is a step in the ["clean-up" ticket](https://app.zenhub.com/workspaces/simplereport-better-data-team-605b43d51ea9f700119a48ee/issues/cdcgov/prime-simplereport/2974) for this feature, but I believe we can remove it cleanly here.

Reply:
Done in f6323fd.